orbit/cmd/desktop/desktop.go

```diff
@@ -122,6 +123,7 @@ func main() {
 
 			tr := http.DefaultTransport.(*http.Transport)
 			if os.Getenv("FLEET_DESKTOP_INSECURE") != "" {
+				// #nosec
```

Comment:
@GuillaumeRoss FYI: Adding exception because running Orbit/Fleet-Desktop with the  env variable `FLEET_DESKTOP_INSECURE` set to `1` is only used in local setups.